# Supplementary Material for James D Galbraith's thesis "The Evolution and Adaptive Effects of Transposable Elements in Birds and Elapids"

The supplementary information for each chapter can be found in the respective folder

## Chapter 1

SI Table 1: Genome assemblies used throughout this analysis. All genomes were downloaded from GenBank.

SI Table 2: Reclassification of previously described full length avian CR1s based on their position within our CR1 phylogeny (SI Figure 1; same color coding).

SI Table 3: Resolution of presence or absence of orthologous CR1 insertions between 10and 60bp in related species in waterfowl, shorebirds, perching birds, parrots, owls, and kiwis + cassowary + emu genomes. Cells highlighted in yellow are the values used to construct Figures and 5 and SI Figure 3.

SI Data 1: Coordinates of full length CR1s identified in each genome in BED format. For the appropriate genome version see SI Table 1.

SI Data 2: Multiple sequence alignment used to create the CR1 phylogeny (SI Figure 1) and Newick tree of said phylogeny.

SI Data 3: Divergence plots of 3â€™ anchored CR1s identified in each species of bird belonging to orders in which we detected full length CR1s. CR1s were identified using a reciprocal BLAST search based on libraries consisting of Repbase avian and crocodilian repeats and the centroids of full length sequences identified within the order clustered in VSEARCH. Jukes-Cantor distance was calculated from the reciprocal BLAST search output.


## Chapter 2

SI Dataset 1: Annotation of HT LINEs  outputusing CENSOR 4.2.2in combination with RepBase.

SI Dataset 2: Megablast 2.7.1 hits for all 7 repeats in 11 sea snake transcriptomes from Crowe-Riddell, 2019 (Molecular Ecology). Results below 250bp in length and <95% identity discarded.

SI Dataset 3: Latin species names and versions of all public genomes used. All were downloaded from GenBank.

SI Dataset 4: Number of repeats similar to Rex1-Snek_1H1, Rex1-Snek_1H2, Rex1-Snek_1H3, Rex1-Snek_2, RTE-Snek_1, RTE-Snek_2, RTE-Kret and Proto2-Snek found in all species search using BLASTN+ 2.7.1 with relaxed parameters.

SI Dataset 5: Sequences of the 7 horizontally transferred LINEs found in _Aipysurus laevis_ and the one horizontally transferred LINE found in _Laticauda colubrina_.

SI Dataset 6: Sequences of all manually curated Rex1s, RTEs and Proto2s from all species.

SI Dataset 7: Rex1 newick tree. Phylogeny created using RAxML from a multiple sequence alignment of the four Rex1s found in _Aipysurus laevis_, all manually curated Rex1s and all Rex1s from RepBase generated using MAFFT and trimmed using Gblocks.

SI Dataset 8: Newick tree of all RTE-like LINEs. Phylogeny created using RAxML from a multiple sequence alignment of Proto2-Snek, RTE-Snek_1, RTE-Snek_2, RTE-Kret, all manually curated Rex1s and all Rex1s from RepBase. generated using MAFFT and trimmed using Gblocks.

SI Dataset 9: A list of the species TimeTree was unable to resolve, and species it used as substitutes nodes for those queried but absent from the TimeTree database.


## Chapter 3

SI Table 1: _Laticauda colubrina_ and _Laticauda laticaudata_ genes with _Harbinger-Snek_ insertions into or overlapping open reading frames, and any noticeable effects on insertion noted from transcript data. Gene coordinates predicted with Liftoff using the RefSeq _Notechis scutatus_ assembly and gene annotation as reference. Repeat annotation performed with RepeatMasker using a custom repeat library (see Methods). Intersect performed using BEDTools. Transcripts mapped to the genome assembly using STAR and viewed in IGV.

SI Table 2: Biological processes with an over/under-representation of _Harbinger-Snek_ insertions into _Laticauda colubrina_ genes. Representation test performed using PANTHER. Gene coordinates predicted with Liftoff using the RefSeq _Notechis scutatus_ assembly and gene annotation as reference. Repeat annotation performed with RepeatMasker using a custom repeat library (see Methods). Intersect performed using plyranges.

SI Table 3: Molecular functions with an over/under-representation of _Harbinger-Snek_ insertions into _Laticauda colubrina_ genes. Representation test performed using PANTHER. Gene coordinates predicted with Liftoff using the RefSeq _Notechis scutatus_ assembly and gene annotation as reference. Repeat annotation performed with RepeatMasker using a custom repeat library (see Methods). Intersect performed using plyranges.

SI Table 4: Biological processes with an over/under-representation of _Harbinger-Snek_ insertions into potential regulatory regions of _Laticauda colubrina_ genes. Representation test performed using PANTHER . Gene coordinates predicted with Liftoff using the RefSeq _Laticauda colubrina_ assembly and gene annotation as reference. Repeat annotation performed with RepeatMasker using a custom repeat library (see Methods). Intersect performed using plyranges.

SI Table 5: Molecular functions with an over/under-representation of _Harbinger-Snek_ insertions into potential regulatory regions of _Laticauda colubrina_ genes. Representation test performed using PANTHER . Gene coordinates predicted with Liftoff using the RefSeq _Laticauda colubrina_ assembly and gene annotation as reference. Repeat annotation performed with RepeatMasker using a custom repeat library (see Methods). Intersect performed using plyranges.

SI Table 6: Latin species names and versions of all public genomes used. All were downloaded from RefSeq when available, else from GenBank.


## Chapter 4

SI Table 1: Genome assembly statistics. Repeat composition was calculated using RepeatMasker and a custom library of curated RepeatModeler libraries and previously described lepidosaur TEs from the Repbase RepeatMasker library.

SI Table 2: A list of the metazoan genome assemblies searched for HTT. All genomes were downloaded from NCBI RefSeq and GenBank.

SI Table 3: Horizontally transferred TEs and the number of hits in species they were identified in. TEs were identified using BLASTN (-task dc-megablast).